pkg/initializer/validator/types.go

```diff
@@ -34,14 +36,18 @@ const (
 	DataDirectoryCorrupt
 	// DataDirectoryStatusUnknown indicates validator failed to check the data directory status.
 	DataDirectoryStatusUnknown
-	// RevisionConsistencyError indicates current etcd revision is inconsistent with latest snapshot revision.
+	// EtcdRevisionConsistencyError indicates current etcd revision is inconsistent with latest snapshot revision.
+	EtcdRevisionConsistencyError
```

Comment:
If I remember right, we discussed that we use only `RevisionConsistencyError` as the returned status, so there's no need for a new error type here.

Reply:
Yes, but I thought a new error will be helpful in future if somebody want to debug .

Reply:
I agree with Shreyas to re-use the existing `RevisionConsistencyError` rather than create a new error type. The distinction between DB file revision being older than backup and the WAL file revision being older than backup is only a _technical_ distinction because of the way we are optimizing the consistency check (checking DB file for a quick check and doing the WAL check only to be doubly sure that the inconsistency is real). It is not a functional or real distinction.